schemas/ispyb/stored_programs/sp_retrieve_grid_info_for_dc.sql

```diff
@@ -0,0 +1,27 @@
+DELIMITER ;;
+CREATE PROCEDURE `retrieve_grid_info_for_dcg`(IN p_dcgId int unsigned)
```

Comment:
Looks like the wrong procedure, but you seem to have the right one in `routines.sql`.